marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -341,6 +353,82 @@ function getEntityTitles() {
     return cts.search(cts.collectionQuery("http://marklogic.com/entity-services/models")).toArray().map(e => e.xpath("//info//title"));
 }
 
+function getFullFlow(flowName, artifactVersion = 'latest') {
```

Comment:
This looks good, but shouldn't this go into artifacts/flows.sjs since it's flow-specific? 

Also @ryanjdew - do we really get any benefit from caching artifacts? That's only good for the duration of the transaction, right? I doubt we need any caching in this function.

Reply:
getFullFlow() has to fetch 'loadData', 'mapping', 'matching' artifacts and settings to build full flow. core.sjs provides a generic interface regardless of artifact type to access them(getArtifact, getArtifactSettings), otherwise if this is in flow.sjs, I will have to include each and every module('loadData'.sjs', 'mapping.sjs' etc) and have multiple 'if' conditions for each type of artifact. It seems better to add it here

Reply:
Removed caching for fullFlow()